ring_of_fire_bot/model/ring.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy import Column, Integer
+
+from ring_of_fire_bot.model.database import Base
+
+
+class Ring(Base):
+    __tablename__ = 'ring'
+    ring_id = Column(Integer, primary_key=True)
+    status = Column(Integer, default=0)
+
+    def __init__(self, status=0):
```

Comment:
this status var should be STATUS class from ring_status.py right?
Can we use the enum in code as much as possible, only converting it to an Integer for the database? Should we use IntEnum? 
```suggestion
    def __init__(self, status=STATUS.WAITING_ON_PARTICIPANTS):
```